src/persist-client/src/lib.rs

```diff
@@ -241,7 +242,7 @@ impl PersistConfig {
             compaction_enabled: !compaction_disabled,
             compaction_heuristic_min_inputs: 8,
             compaction_heuristic_min_updates: 1024,
-            consensus_connection_pool_max_size: 8,
+            consensus_connection_pool_max_size: 50,
```

Comment:
@benesch wanted to get a gut check on this one w.r.t. Cockroach. if I run `environmentd` locally, it will actually use the full 50 (it'll actually go to 75+ if unbounded... not entirely sure why yet)

Reply:
I'm 👍🏽 on trying it and seeing what happens. It matches my intuition for what should be acceptable with Cockroach—these connections are just a cheap goroutine—but the Cockroach docs caution against this for reasons I don't totally understand.